src/StakeRegistryStorage.sol

```diff
@@ -42,6 +42,8 @@ abstract contract StakeRegistryStorage is IStakeRegistry {
      * corresponding multipliers for that specific quorum
      */
     mapping(uint8 => StrategyParams[]) public strategyParams;
+    mapping(uint8 => IStrategy[]) public strategiesPerQuorum;
```

Comment:
gotta reduce the storage gap size by 1 if we add this new mapping

Reply:
we really need to add some automated checking for that after we do mainnet

Reply:
https://github.com/Layr-Labs/eigenlayer-middleware/pull/170/commits/3688a02f4a7841f2d3befbdd3821cc3b4d22c602#diff-3aca207bfe24bd32c66e23f704da75c45051c4fad2b745aa2efaf937e9dc6e2dR58